<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sorter</class>
 <widget class="QMainWindow" name="Sorter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sorter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sorting Algorithm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="chosenAlgorithm">
        <item>
         <property name="text">
          <string>Bubble Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Selection Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Insertion Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Merge Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quick Sort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heap Sort</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Number Of Columns</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="columnCount">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Delay(ms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delay">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="sortBtn">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopBtn">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shuffleBtn">
        <property name="text">
         <string>Shuffle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
